openpype/hosts/tvpaint/api/lib.py

```diff
@@ -50,8 +50,7 @@ def parse_layers_data(data):
             "group_id": int(group_id),
             "visible": visible == "ON",
             "position": int(position),
-            # Opacity from 'tv_layerinfo' is always set to '0' so it's unusable
-            # "opacity": int(opacity),
+            "opacity": int(get_layer_opacity(layer_id)),
```

Comment:
In my opinion the function should return an int not a string:
```python
"opacity": get_layer_opacity(layer_id),
```

Reply:
yes !